src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -1371,17 +1445,20 @@ public int doFinal(byte[] in, int inOfs, int inLen, byte[] out,
             }
 
```

Comment:
Unrelated to your change, but I noticed that the if-block from line 1443-1445 should be moved up before the ArrayUtil.nullAndBoundsCheck(...), otherwise the 3rd argument may be negative and lead to a wrong exception being thrown.